README.md

```diff
@@ -1,15 +1,126 @@
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+    mailchimp_database: your_database_name
+    mailchimp_schema: your_schema_name 
+```
+
+
+### Passthrough Columns
+
+This package includes all of the source columns that are defined in the macros folder. We recommend including custom columns in this package because the staging models only bring in the standard columns for the `MEMBER` table.
+
+You can add more columns using our passthrough column variables. These variables allow the passthrough columns to be aliased (`alias`) and casted (`transform_sql`) if you want, although it is not required. You can configure datatype casting by using a SQL snippet within the `transform_sql` key. You may add the desired SQL snippet while omitting the `as field_name` part of the casting statement - we rename this column with the alias attribute - and your custom passthrough columns will be casted accordingly.
+
+Use the following format for declaring the respective passthrough variables (note to call the alias name, not original name, in the transform_sql):
+
+```yml
+# dbt_project.yml
+
+vars:
+
+  mailchimp__members_pass_through_columns:
+    - name:           "custom_field_name"
+      alias:          "normal_field_name"
+      transform_sql:  "cast(normal_field_name as string)"
+```
+
+
+### Disabling models
+
+It's possible that your Mailchimp connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Mailchimp or actively excluded some tables from your syncs. To disable the corresponding functionality in the package, you must add the relevant variables. By default, all variables are assumed to be `true`. Add variables for only the tables you would like to disable:  
+
+```yml
+# dbt_project.yml
+
+...
+config-version: 2
+
+vars:
+    usting_automations: false   # Disable if you are not using automations 
+    
+```
+
+### Changing the Build Schema
+By default this package will build the Mailchimp staging models within a schema titled (<target_schema> + `_stg_mailchimp`) in your target database. If this is not where you would like your Mailchimp staging data to be written to, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+models:
+    mailchimp_source:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+```
+
+### Changing the Build Schema
+
+By default, this package builds the Pendo staging models within a schema titled (`<target_schema>` + `_stg_mailchimp`) in your target database. If this is not where you would like your Mailchimp staging data to be written to, add the following configuration to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+
+...
+models:
+    mailchimp_source:
+        +schema: my_new_schema_name # leave blank for just the target_schema
+```
+
```

Comment:
Looks like this is a duplicate and holdover from the Pendo copy. Can you remove this.